### ISSUE TITLE
When creating a “final” release, delete the corresponding “pre-release”

### ISSUE THREAD

**Issue by aviflax** [FundingCircle/fc4-framework#203]
The [list of releases](https://github.com/FundingCircle/fc4-framework/releases) is cluttered with obsolete pre-releases. We should consider deleting the pre-releases once the change is merged into master and a final release is published. We could do this in the CI job that publishes the final release.

**Comment by aviflax** [comment]
I’ve been thinking this over and I think it might be easier to create a schedule job that would delete all pre-releases older than a certain age. TBD.
